### PR TITLE
migrate tests from tape to jest

### PR DESCRIPTION
**Pull request by axe312ger**
* [x] rewrite tests
* [x] move to codecov
* [x] increase coverage / add new tests

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -170,7 +169,7 @@ export default function runContentfulExport (params) {
         console.log('No data was exported')
       }
 
-      if ('assetDownloads' in summary) {
+      if ('assetDownloads' in ctx) {
```
Comment (axe312ger) [inline lib/index.js]: A small bugfix I sneaked in. The assets table actually was never shown. Now it does :)

File: package.json [hunk 2]
```diff
@@ -15,12 +15,12 @@
...
-    "pretest": "npm run lint",
-    "test": "npm run test:cover && npm run test:integration",
-    "test:integration": "npm run build && rimraf ./test/integration/tmp && BABEL_ENV=test babel-node ./test/runner-integration",
-    "test:cover": "BABEL_ENV=test babel-node ./node_modules/istanbul/lib/cli.js cover test/runner-unit",
-    "test:only": "BABEL_ENV=test babel-node ./test/runner-unit",
-    "test:debug": "BABEL_ENV=test babel-node debug ./test/runner-unit",
+    "pretest": "npm run lint && npm run build && rimraf ./test/integration/tmp",
+    "test": "npm run test:unit && npm run test:integration",
+    "test:unit": "jest --testPathPattern=test/unit --coverage",
+    "test:unit:debug": "npm run test:unit -- --watch",
```
Comment (Khaledgarbaya) [inline package.json]: Does this replace the `node inspect` ?

Comment (axe312ger) [reply package.json]: Hmm nope, it doesn't. I will try to find a solution.

Comment (axe312ger) [reply package.json]: done :)

**Pull request merged**